shell/common/asar/archive.h

```diff
@@ -65,10 +67,11 @@ class Archive {
   int GetFD() const;
 
   base::FilePath path() const { return path_; }
-  base::DictionaryValue* header() const { return header_.get(); }
 
  private:
-  base::FilePath path_;
+  // Lock protects |external_files_|
+  base::Lock lock_;
```

Comment:
Let's put this down next to `external_files_`, and maybe name it `external_files_lock_` if that's the only value it protects?